### PR TITLE
*Draft* Front End UI

### PR DESCRIPTION
**Pull request by AdnanAhsanNYCity**
Checking Merge Conflicts ***DRAFT.***

### CODE REVIEW

**Comment by TheHarbinger99** [comment]
Looks good. 
Just one question for the future if time permits, can the code for displaying reviews in Google map container be refactored into a function and called later 5 times or not. I am curious to know. 
Other than that , 
Lgtm

File: backend/frontend/src/styles/Form.css [hunk 1]
```diff
@@ -120,6 +120,89 @@ button[type="button"] {
...
+  font-size: 16px;
+  margin: 4px 1px;
+  cursor: pointer;
+  display: flex;
+  position: relative;
+  left: 180px;
+  bottom: 85px;
+}
+
+.buttonaddreview {
```
Comment (TheHarbinger99) [inline backend/frontend/src/styles/Form.css:191]: These lines are repeated as lines 159-172 above. The .buttonaddreview part. Can you check on this once

**Review by TheHarbinger99 — APPROVED** [review]
Please check some reviews and comments I made.
Lgtm

File: backend/frontend/src/styles/Form.css (hunk 1 above)
Comment (AdnanAhsanNYCity) [reply backend/frontend/src/styles/Form.css]: Yep! Removed, was just extra dead code.

**Review by as14680 — APPROVED** [review]
lgtm

**Review by TheHarbinger99 — APPROVED** [review]
lgtm

**Review by as14680 — APPROVED** [review]
lgtm

**Review by TheHarbinger99 — APPROVED** [review]
lgtm

**Review by ds6332 — APPROVED** [review]

**Pull request merged**